src/components/MapNode.tsx

```diff
@@ -38,9 +38,10 @@ export const MapNode = observer((props) => {
   const terms = getMapNodeTerms(currentRevision) || []
 
   // Children
-  const hasChildren = Object.keys(nodeChildren).length > 0
-  const childKeys = childrenOrder || Object.keys(nodeChildren)
+  const hasChildren = children && Object.keys(children).length > 0
+  const childrenKeys = childrenOrder || (children && Object.keys(children))
```

Comment:
lodash's `_.keys` function will work without the check if it exists btw. I often import lodash functions like `import keys from 'lodash/keys'` because it doesn't import the whole package that way.